docs/howto/virtual/postfix.md

```diff
@@ -36,7 +36,6 @@ This allows individual /SEEN information for the public namespace.
 mail_driver = maildir
 mail_path = ~/maildir
 mail_plugins {
-  acl = yes
```

Comment:
This (and other ACL related items) do not appear to belong to this commit. I'd split them off into another one, even if then it only contains two changes.